sale_order_digitized_signature/models/sale.py

```diff
@@ -7,9 +7,7 @@
 class SaleOrder(models.Model):
     _inherit = 'sale.order'
 
-    customer_signature = fields.Binary(
-        string='Customer acceptance',
-    )
+    customer_signature = fields.Binary(string='Customer acceptance')
```

Comment:
Maybe we can put this as `attachment=True` (it will require migration scripts) for better DB space usage.

Reply:
Ping @mgosai